showcase-docs/dynamic-plugins.md

```diff
@@ -243,16 +251,26 @@ In order to do this, just add the following dynamic plugins to the `global.dynam
 global:
   dynamic:
     plugins:
-      - package: '@dfatwork-pkgs/scaffolder-backend-module-http-request-wrapped-dynamic'
-      - package: '@dfatwork-pkgs/plugin-events-backend-module-test-dynamic'
-      - package: '@dfatwork-pkgs/explore-backend-wrapped-dynamic'
+      - package: '@dfatwork-pkgs/scaffolder-backend-module-http-request-wrapped-dynamic@4.0.9-0'
+        # Integrity can be found at https://registry.npmjs.org/@dfatwork-pkgs/scaffolder-backend-module-http-request-wrapped-dynamic
+        integrity: 'sha512-+YYESzHdg1hsk2XN+zrtXPnsQnfbzmWIvcOM0oQLS4hf8F4iGTtOXKjWnZsR/14/khGsPrzy0oq1ytJ1/4ORkQ=='
+      - package: '@dfatwork-pkgs/plugin-events-backend-module-test-dynamic@0.0.1'
+        # https://registry.npmjs.org/@dfatwork-pkgs/plugin-events-backend-module-test-dynamic
+        integrity: 'sha512-YaOmijWWWZqlNubQnpiaHwtZNlELzE2av2kkuzeLPg4YaupnaVTdXsR71d4uz6bfqA8QOYi9o4sJD4cJivE6jA=='
+      - package: '@dfatwork-pkgs/explore-backend-wrapped-dynamic@0.0.9-next.11'
+        # https://registry.npmjs.org/@dfatwork-pkgs/explore-backend-wrapped-dynamic
+        integrity: 'sha512-/qUxjSedxQ0dmYqMWsZ2+OLGeovaaigRRrX1aTOz0GJMwSjOAauUUD1bMs56VPX74qWL1rf3Xr4nViiKo8rlIA=='
         pluginConfig:
           proxy:
             endpoints:
               /explore-backend-completed:
                 target: 'http://localhost:7017'
-      - package: '@dfatwork-pkgs/search-backend-module-explore-wrapped-dynamic'
-      - package: '@dfatwork-pkgs/plugin-catalog-backend-module-test-dynamic'
+      - package: '@dfatwork-pkgs/search-backend-module-explore-wrapped-dynamic@0.1.3-next.1'
+        # https://registry.npmjs.org/@dfatwork-pkgs/search-backend-module-explore-wrapped-dynamic
+        integrity: 'sha512-mv6LS8UOve+eumoMCVypGcd7b/L36lH2z11tGKVrt+m65VzQI4FgAJr9kNCrjUZPMyh36KVGIjYqsu9+kgzH5A=='
+      - package: '@dfatwork-pkgs/plugin-catalog-backend-module-test-dynamic@0.0.0'
```

Comment:
Well, these are a temporary thing, as mentioned here: https://github.com/janus-idp/backstage-showcase/blob/555c58795c825ce47386ccb79f81de01a1f3784a/showcase-docs/dynamic-plugins.md#warning.

As soon as we have a bunch of real `backstage-plugins` plugins made dynamic-enabled, we would probably change this to point to more standard plugins, instead of those of my initial demo.